package.json

```diff
@@ -1,31 +1,40 @@
 {
-    "devDependencies": {
-      "husky": "^0.14.3",
-      "codecov": "^2.2.0",
-      "jest": "^21.2.1",
-      "lerna": "^2.3.1",
-      "lint-staged": "^4.2.3",
-      "prettier-eslint-cli": "^4.4.0"
-    },
-    "scripts": {
-      "postinstall": "lerna bootstrap",
-      "coverage": "codecov",
-      "clean": "lerna clean",
-      "updated": "lerna updated",
-      "test": "jest",
-      "publish": "lerna publish --exact",
-      "precommit": "lint-staged"
-    },
-    "lint-staged": {
-      "*.js": [
-        "prettier-eslint --write",
-        "jest --findRelatedTests",
-        "git add"
-      ]
-    },
-    "jest": {
-      "rootDir": "packages",
-      "coverageDirectory": "./coverage/",
-      "collectCoverage": true
-    }
-  }
\ No newline at end of file
+  "scripts": {
+    "postinstall": "lerna bootstrap",
+    "coverage": "codecov",
+    "clean": "lerna clean",
+    "updated": "lerna updated",
+    "test": "jest",
+    "publish": "lerna publish --exact",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier-eslint --write",
+      "jest --findRelatedTests",
+      "git add"
+    ]
+  },
+  "jest": {
+    "rootDir": "packages",
+    "coverageDirectory": "./coverage/",
+    "collectCoverage": true
+  },
+  "dependencies": {
+    "async": "^2.5.0",
```

Comment:
Long: One rule we try to follow is that the root package.json should be dependencies free, and only have devDependencies), the opposite apply for inner packages: they should be devDependencies free as their devDependencies could be simply moved to the devDependencies of the root manifest.
In this way the root work as the orchestrator for everything needed for dev and publishing, while each package is simply let to handle only its own dependencies. A nice thing with this is that you have consistency devDependencies across the whole monorepo.

TLDR: don't put those dependencies here, but only on the packages that needs them